modules/consensus.go

```diff
@@ -87,6 +87,16 @@ type ConsensusSet interface {
 	// This is a thread-safe way of managing updates.
 	ConsensusSetSubscribe(ConsensusSetSubscriber)
 
+	// CurrentBlock returns the most recent block on the heaviest fork known to
+	// the consensus set.
+	CurrentBlock() types.Block
```

Comment:
Wasn't this why you needed to subscribe the API server?
I guess you need the height as well.